File: pingyougou-manager-web/src/main/webapp/js/services/brandService.js
app.service("brandService",function($http){
			//查询品牌列表
			this.findAll=function(){
				return $http.get('../brand/findAll.do');
			}
			//查询品牌列表
			this.findPage=function(page,size){
				return $http.get('../brand/findPage.do?page='+page+'&size='+size);
			}
			//增加
			this.add=function(entity){
				return $http.post('../brand/add.do',entity);
			}
			//修改
			this.update=function(entity){
				return $http.post('../brand/update.do',entity);
			}
			//查询一个
			this.findOne=function(id){
				return $http.get('../brand/findOne.do?id='+id);
			}
			//删除
			this.dele=function(ids){
				return $http.get('../brand/delete.do?ids='+ids);
			}
			//多条件查询
			this.search=function(page,size,searchEntity){
				return $http.post('../brand/search.do?page='+page+'&size='+size,searchEntity);
			}
			//下拉列表查询
			this.selectOptionList=function(){
				return $http.get('../brand/selectOptionList.do');
			}
		})

File: pingyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

/**
 * 品牌列表
 * @author ASUS
 *
 */
public interface BrandService {
	
	/**
	 * 查询所有
	 * @return
	 */
	public List<TbBrand> findAll();
	/**
	 * 品牌下拉框数据
	 * @return
	 */
	public List<Map> selectOptionList();
	/**
	 * 分页
	 * @param pageNum 当前页码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 增加方法
	 * @param tbBrand
	 */
	public void add(TbBrand tbBrand);
	/**
	 * id查一个
	 * @param id
	 * @return
	 */
	public TbBrand findOne(long id);
	/**
	 * 修改
	 * @param tbBrand
	 */
	public void update(TbBrand tbBrand);
	
	/**
	 * 删除
	 * @param ids
	 */
	public void delete(long[] ids);
	/**
	 * 条件查询
	 * @param tbBrand
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);
}
